docker/README.md

```diff
@@ -79,19 +79,19 @@ To pull the a particular container, run the following command
 #### Pulling the latest tag:
 Docker pull by default pulls the latest tag. This tag isn't available until after an official release. 
 ```bash
-docker pull awsdeeplearningteam/mxnet-model-server # This gets the latest release which is the same as 1.0.0-mxnet-cpu
+docker pull awsdeeplearningteam/mxnet-model-server # This gets the latest release which is the same as 1.0.1-mxnet-cpu
 ``` 
 
-#### Pulling the `1.0.0-mxnet-cpu` tag:
+#### Pulling the `1.0.1-mxnet-cpu` tag:
 To pull a official 1.0 MMS CPU container run the following command. This tag isn't available until after an official release. 
 ```bash
-docker pull awsdeeplearningteam/mxnet-model-server:1.0.0-mxnet-cpu 
+docker pull awsdeeplearningteam/mxnet-model-server:1.0.1-mxnet-cpu 
 ``` 
 
-#### Pulling the `1.0.0-mxnet-gpu` tag:
+#### Pulling the `1.0.1-mxnet-gpu` tag:
 To pull a official 1.0 MMS GPU container run the following command. This tag isn't available until after an official release. 
 ```bash
-docker pull awsdeeplearningteam/mxnet-model-server:1.0.0-mxnet-gpu 
+docker pull awsdeeplearningteam/mxnet-model-server:1.0.1-mxnet-gpu 
```

Comment:
better to have a:
docker pull awsdeeplearningteam/mxnet-model-server:latest-mxnet-gpu

Reply:
Agreed. This is the same work as above. Lets make this after 1.0.2 release